public/endi.css

```diff
@@ -0,0 +1,78 @@
+body {
+    background-color: #121212;
+    text-align: center;
+    color: #f0f0f0;
+    font-family: 'Arial', sans-serif;
+}
+
+#navbar {
+    background-color: #1e1e1e;
+    padding: 10px;
+}
+
+#navbar ul {
+    list-style-type: none;
+    padding: 0;
+}
+
+#navbar li {
+    display: inline;
+    margin: 0 15px;
+}
+
+#navbar a {
+    color: #00bfff;
+    text-decoration: none;
+    font-weight: bold;
+}
+
+#navbar a:hover {
+    text-decoration: underline;
+}
+.my-image {
```

Comment:
Emertim shume i mire.
Nese don me kqyr ma detajisht per emertim te css classave "https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/"